notebooks/visualization/Cartopy-Intro.ipynb

```diff
@@ -0,0 +1,389 @@
+    "ax.add_feature(cfeature.BORDERS, linestyle=':')\n",
+    "ax.add_feature(cfeature.STATES, linestyle=':')\n",
+    "ax.add_feature(cfeature.LAKES, alpha=0.5)\n",
+    "ax.add_feature(cfeature.RIVERS, edgecolor='tab:green')\n",
+    "\n",
+    "# Set the extent\n",
+    "ax.set_extent([-130, -60, 20, 55])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The map features are available at several different scales depending on how large the area you are covering is. The scales can be accessed using the `with_scale method`. Natural Earth features are available at 110m, 50m and 10m."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(10, 8))\n",
+    "ax = fig.add_subplot(1, 1, 1, projection=ccrs.LambertConformal())\n",
+    "\n",
+    "# Add variety of features\n",
+    "ax.add_feature(cfeature.LAND)\n",
+    "ax.add_feature(cfeature.OCEAN)\n",
+    "ax.add_feature(cfeature.COASTLINE)\n",
+    "\n",
+    "# Can also supply matplotlib kwargs\n",
+    "ax.add_feature(cfeature.BORDERS.with_scale('50m'), linestyle=':')\n",
+    "ax.add_feature(cfeature.STATES.with_scale('50m'), linestyle=':')\n",
+    "ax.add_feature(cfeature.LAKES.with_scale('50m'), alpha=0.5)\n",
+    "ax.add_feature(cfeature.RIVERS.with_scale('50m'), edgecolor='tab:green')\n",
+    "\n",
+    "ax.set_extent([-130, -60, 20, 55])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Interested in other map features? Check out the documentation from the [Natural Earth Project](http://www.naturalearthdata.com/)\n",
+    "\n",
+    "### US County Boundaries\n",
+    "\n",
+    "MetPy also has US County boundaries available at 20m, 5m, and 500k resolutions. Checkout the example below to see the difference between different resolutions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from metpy.plots import USCOUNTIES"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proj = ccrs.LambertConformal(central_longitude=-85.0, central_latitude=45.0)\n",
+    "\n",
+    "fig = plt.figure(figsize=(12, 9))\n",
+    "ax1 = fig.add_subplot(1, 3, 1, projection=proj)\n",
+    "ax2 = fig.add_subplot(1, 3, 2, projection=proj)\n",
+    "ax3 = fig.add_subplot(1, 3, 3, projection=proj)\n",
+    "\n",
+    "for scale, axis in zip(['20m', '5m', '500k'], [ax1, ax2, ax3]):\n",
+    "    axis.set_extent([270.25, 270.9, 38.15, 38.75], ccrs.Geodetic())\n",
+    "    axis.add_feature(USCOUNTIES.with_scale(scale), edgecolor='black')\n",
+    "    axis.set_title(scale)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---\n",
+    "## 3. Plotting data"
```

Comment:
Maybe add one quick sentence about what you're about to do in this code block (Ie. "In addition to standard known data sets, we can also plot our own data on a map.  Here we're going to plot a point at....").

Ah yes, especially since the next markdown block actually tells why it failed (we need to know what we were expecting it to do first... ie, plot a point at Boulder, CO).